src/client/search_requests.py

```diff
@@ -984,19 +933,16 @@ class CompareFields:
 OperationSortItems = BaseSortItems.for_fields(
     'OperationSortItems', ['id', 'submitted', 'finished'],
     # docstring
-    """Parameters for sorting operations search results
-
-    You can specify multiple parameters.
-    To change the sorting direction (sort in descending order), add a hyphen before the parameter. For example, sort=-id.
+    """Keys for sorting operations in search results.
 
     Attributes:
-        items: Fields by which to sort. Possible values:
-            * id - Subscription ID.
+        items: A list of sorting keys. Supported values:
+            * id - Operation ID.
```

Comment:
Shouldn't these items be also highlighted like those in lines 888-890, for example? With backticks and single quotes?